bindings/generator/api/platform.py

```diff
@@ -3,12 +3,13 @@
 from .common import Enum, EnumItemUnit
 
 
-class OS(Enum):
+class Platform(Enum):
     Linux = EnumItemUnit
     MacOs = EnumItemUnit
```

Comment:
Funny to have `OS` but not `MacOS`

Reply:
I'm not sure to understand, do you want to rename `MacOs` to `MacOS` ?

Reply:
done